LevelUp.Pos.ProposedOrders/ProposedOrderCalculator.cs

```diff
@@ -88,23 +94,45 @@ int customerPaymentAmount
             int appliedDiscountAmount
         )
         {
-            AdjustedCheckValues values = CalculateCreateProposedOrderValues(
-                totalOutstandingAmount,
+            int totalOutstandingAmountWithDiscount = totalOutstandingAmount + Math.Abs(appliedDiscountAmount);
+
+            int adjustedSpendAmount = CalculateAdjustedCustomerPaymentAmount(
+                customerPaymentAmount, 
+                totalOutstandingAmountWithDiscount);
+
+            int adjustedTaxAmount = CalculateAdjustedTaxAmount(
+                totalOutstandingAmountWithDiscount, 
```

Comment:
I think totalOutstandingAmountWithDiscount is the wrong metric for calculating tax and exemption.

Consider this contrived example:
 
$25 on a check, of which $10 is the item cost and $15 is tax
The customer would like to pay $11 and has $8 in discount credit.

What is the adjusted tax?  Let's discuss in person